python/paddle/fluid/tests/unittests/mkldnn/test_matmul_v2_mkldnn_op.py

```diff
@@ -23,14 +24,12 @@
 import paddle.fluid as fluid
 import paddle.fluid.framework as framework
 
-paddle.enable_static()
 
-
-def reference_matmul(X, Y, transpose_X=False, transpose_Y=False):
+def reference_matmul(X, Y, trans_x=False, trans_y=False):
```

Comment:
Why do you change transpose_x to trans_x ? I feel keeping code self explaining is good. With trans_x, if somebody remove below comments,  may get confused with trans_x. 

Just my thought.